src/helm/proxy/clients/auto_client.py

```diff
@@ -202,6 +202,7 @@ def _get_tokenizer_client(self, tokenizer: str) -> Client:
                 "google",
                 "gooseai",
                 "huggingface",
+                "meta-llama",
```

Comment:
Should this be llama2?

Reply:
Nope, the Hugging Face organization name is `meta-llama` even though the display name is "Meta Llama 2"... https://huggingface.co/meta-llama/